megatron/model/gpt2_model.py

```diff
@@ -227,20 +220,19 @@ def _logits_helper(embedding, lm_output):
                 lm_output,
                 embedding.word_embeddings_weight,
                 self.parallel_output)
-        
+
         if weight_tying:
             self.specs.append(
                 TiedLayerSpec('embed',
-                            EmbeddingPipe,
-                            self.hidden_size,
-                            args.padded_vocab_size,
-                            args.max_position_embeddings,
-                            args.hidden_dropout,
-                            self.init_method,
-                            self.num_tokentypes,
-                            args.sinusoidal_pos_emb,
-                            forward_fn=_logits_helper,
-                            tied_weight_attr='word_embeddings_weight')
+                              EmbeddingPipe,
+                              self.hidden_size,
+                              args.padded_vocab_size,
+                              args.max_position_embeddings,
+                              args.hidden_dropout,
+                              self.init_method,
+                              self.num_tokentypes,
+                              forward_fn=_logits_helper,
+                              tied_weight_attr='word_embeddings_weight')
```

Comment:
Is here a `args.pos_emb == "sinusoidal"` missing instead of the previous `args.sinusoidal_pos_emb`?